srcpkgs/eiskaltdcpp-qt/template

```diff
@@ -1,19 +1,20 @@
 # Template file for 'eiskaltdcpp-qt'
 pkgname=eiskaltdcpp-qt
-version=2.2.10
-revision=11
+version=2.4.1
+revision=1
 wrksrc="eiskaltdcpp-${version}"
 build_style=cmake
-configure_args="-DUSE_MINIUPNP=ON -DUSE_QT5=ON -DUSE_QT=OFF"
+configure_args="-DUSE_MINIUPNP=ON -DUSE_QT5=ON -DUSE_QT=OFF -DUSE_ASPELL=OFF"
 hostmakedepends="gettext pkg-config qt5-tools-devel qt5-host-tools qt5-qmake"
-makedepends="bzip2-devel zlib-devel libressl-devel libidn-devel boost-devel
- qt5-devel miniupnpc-devel qt5-multimedia-devel"
-short_desc="EiskaltDC++ is a cross-platform Direct Connect and ADC clent"
-maintainer="samsky72 <samsky72@gmail.com>"
+makedepends="bzip2-devel zlib-devel libressl-devel libidn-devel lua51-devel
```

Comment:
Can we try and use newer lua?

Reply:
`lua52` is the latest supported/tested version now in upstream.

Reply:
Updated build dependency to `lua52`.